imagecraft/pack/gptutil.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2025 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Utility functions for GPT-formatted disks."""
+
+import json
+from pathlib import Path
+from typing import Any, cast
+
+from craft_cli import CraftError, emit
+
+from imagecraft.models import Volume
+from imagecraft.pack import diskutil
+from imagecraft.subprocesses import run
+
+# Supported Sector Sizes
+
+SUPPORTED_SECTOR_SIZES = [512]
```

Comment:
Not strictly necessary, but I like seeing constants be immutable types as it prevents accidental mutation.

```suggestion
SUPPORTED_SECTOR_SIZES = (512,)
```